src/core/managers/auth_manager.py

```diff
@@ -88,26 +88,23 @@ def check_acl(item_id, acl_check, user):
         item_type = "OSINT Source Group"
         allowed = OSINTSourceGroup.allowed_with_acl(item_id, user, check_see, check_access, check_modify)
 
-    if acl_check == ACLCheck.NEWS_ITEM_ACCESS or acl_check == ACLCheck.NEWS_ITEM_MODIFY:
+    if acl_check in [ACLCheck.NEWS_ITEM_ACCESS, ACLCheck.NEWS_ITEM_MODIFY]:
         item_type = "News Item"
         allowed = NewsItem.allowed_with_acl(item_id, user, check_see, check_access, check_modify)
 
-    if acl_check == ACLCheck.REPORT_ITEM_ACCESS or acl_check == ACLCheck.REPORT_ITEM_MODIFY:
+    if acl_check in [ACLCheck.REPORT_ITEM_ACCESS, ACLCheck.REPORT_ITEM_MODIFY]:
         item_type = "Report Item"
         allowed = ReportItem.allowed_with_acl(item_id, user, check_see, check_access, check_modify)
 
-    if acl_check == ACLCheck.PRODUCT_TYPE_ACCESS or acl_check == ACLCheck.PRODUCT_TYPE_MODIFY:
+    if acl_check in [ACLCheck.PRODUCT_TYPE_ACCESS, ACLCheck.PRODUCT_TYPE_MODIFY]:
         item_type = "Product Type"
         allowed = ProductType.allowed_with_acl(item_id, user, check_see, check_access, check_modify)
 
     if not allowed:
         if check_access:
-            log_manager.store_user_auth_error_activity(user,
-                                                       "Unauthorized access attempt to " + item_type + ": " + item_id)
+            log_manager.store_user_auth_error_activity(user, "Unauthorized access attempt to {}: {}".format(item_type, item_id))
```

Comment:
Can't we use format strings? The docker images use python 3.7 (and there are not other requirements documented for non-docker installations).
```suggestion
            log_manager.store_user_auth_error_activity(user, f"Unauthorized access attempt to item_type: item_id")
```